saleor/core/utils/translations.py

```diff
@@ -0,0 +1,28 @@
+from django.utils.translation import get_language
+
+
+class TranslationWrapper(object):
+    def __init__(self, instance, locale):
+        self.instance = instance
+        self.translation = next((
+            t for t in instance.translations.all()
```

Comment:
In this line, we assume that the relation `instance.translations` exists, but it may be undefined if the model with translated fields and reverse relation `translations` was not defined. I think we should add a check here and throw an error in that case (we can implement it later).